### DISCUSSION TITLE
Can this be used without session manager?

### DISCUSSION THREAD

**Discussion by dungle-scrubs** [coffebar/neovim-project#38] [Q&A]
Great plugin, thank you. I like the ability to quickly access any project and recent projects, but I receive too many session manager errors when restoring state (I can submit issues separately when I have time). 

Is it possible to isolate only the project functionality and disable the session saving completely?

**Comment by coffebar** [comment]
Without sessions it will do only 2 things: show the telescope popup window and run "cd" command. Not much, so there is no such option for now. 
If you have a lot of exceptions when restore, search for plugin names in traceback - high chance that some plugins need improvement and it's an easy fix, like null check (file and line can be found in error's trace), so you can contribute.
If an error is not related to plugins, try to change your neovim version, for example try the bliding edge or rollback to previous, that did work better, until somebody else fix this.

UPD: today's [fix](https://github.com/coffebar/neovim-project/commit/901b778533d4025bc235adee5e1421e265a40ee4) possibly resolved your current issue
